src/mlpack/bindings/python/print_type_doc_impl.hpp

```diff
@@ -155,9 +155,13 @@ std::string PrintTypeDoc(
 {
   return "An mlpack model pointer.  This type can be pickled to or from disk, "
       "and internally holds a pointer to C++ memory containing the mlpack "
-      "model.  Note that this means that the mlpack model itself cannot be "
-      "easily inspected in Python; however, the pickled model can be loaded "
-      "in C++ and inspected there.";
+      "model.  This model pointer has 2 methods using which the parameters "
+      "of the model can be inspected as well as changed through Python.  "
+      "The get_cpp_params() method returns a python ordered dictionary that "
+      "contains all the parameters of the model.  The user can inspect the "
+      "parameters as well change the parameter values in the dictionary "
+      "(without deleting any keys) and pass that back into the model "
```

Comment:
```suggestion
      "contains all the parameters of the model.  These parameters can "
      "be inspected and changed.  To set new parameters for a model, "
      "pass the modified dictionary (without deleting any keys) to the "
      "`set_cpp_params()` method."
```
I think maybe we can clarify this part a little bit---let me know what you think. :+1:

(I also didn't check closely that everything here is 80 characters, so it's possible it could be wrapped better.)